### DISCUSSION TITLE
Block Titles

### DISCUSSION THREAD

**Discussion by LeoLDLeo** [brianpetro/obsidian-smart-connections#416] [General]
Hello, I am a Smart Connection Supporter and I love using this plugin to find related notes in my Obsidian vault. However, I am encountering a problem. My daily notes follow the same structure and block names (e.g. ## Thoughts, ## Notes Created Today, etc.). Smart Connection matches other notes with the same structure and returns other notes with the same block names, even if other topics and themes are not quite that related. If I were to remove the block names, then finding related notes becomes a bit easier, but I lose the benefit of having a consistent format for my daily notes.

Is there a way to make Smart Connection ignore the block names and focus on the content of the notes instead? Or the only way is to remove any structure in my daily notes so that Smart Connection can find more relevant matches? I would appreciate any ideas or help from anyone who had the same issue. Thank you.

**Comment by dsearcher** [comment]
I'm also interested in the topic of data preparation and better structuring the notes in the first place.

1) What is the best approach to structuring the notes you would recommend to make smart connections work most efficiently in the current implementation?

2) What is best practices would you recommend to follow when we create a data structure overall in order to prepare to the state when almost all of our data will become available to LLMs and interconnected in all devices that we use? It's probably not that far from today, couple of years of development. @brianpetro

**Reply by brianpetro** [reply]
Hey @dsearcher great questions.

1) the current implementation is still changing regularly. But I try to make Smart Connections compatible with many different note structures.

2) I think what matters most is that the structure makes sense to you. The technology and software will continue to adapt. And without any obvious limitations, the optimal policy would be one that focuses on yourself. 

🌴

**Comment by brianpetro** [comment]
@LeoLDLeo 

This is something that hasn't changed much since the beginning of Smart Connections, but can still use a lot of work.

I recently mentioned some of the technical details for a similar issue here https://github.com/brianpetro/obsidian-smart-connections/issues/527#issuecomment-2025055813

I also have a long list of improvements I want to make to this part of Smart Connections. And while it's currently not top priority, this is a key component to Smart Connections so will be addressed sooner than later.

Thanks for your support and sorry for the late reply!
🌴
